pkg/nsx/services/subnetport/builder.go

```diff
@@ -102,3 +110,61 @@ func getObjectMeta(obj interface{}) *metav1.ObjectMeta {
 func getCluster(service *SubnetPortService) string {
 	return service.NSXConfig.Cluster
 }
+
+func buildSubnetPortExternalAddressBindingFromExisting(subnetPort *model.VpcSubnetPort, existingSubnetPort *model.VpcSubnetPort) *model.VpcSubnetPort {
+	if existingSubnetPort == nil {
+		return subnetPort
+	}
+	if existingSubnetPort.ExternalAddressBinding != nil {
+		if subnetPort.ExternalAddressBinding != nil {
+			// update is not supported
+			subnetPort.ExternalAddressBinding = &model.ExternalAddressBinding{}
```

Comment:
Since "update is not supported", why here it's not `subnetPort.ExternalAddressBinding = existingSubnetPort.ExternalAddressBinding`?

Reply:
Changed to `update is not supported, keep existing ExternalAddressBinding`.